test/Parse/enum.swift

```diff
@@ -552,3 +552,14 @@ enum SE0155 {
   // expected-note@-1 {{did you mean to remove the empty associated value list?}} {{17-18=}}
   // expected-note@-2 {{did you mean to explicitly add a 'Void' associated value?}} {{17-17=Void}}
 }
+
+// SR-11261
+enum SR11261 {
+  case identifier
+  case operator // expected-error {{keyword 'operator' cannot be used as an identifier here}} expected-note {{if this name is unavoidable, use backticks to escape it}} {{8-16=`operator`}} expected-error {{expected pattern}}
```

Comment:
`error: expected pattern` is also irrelevant here.
I'm OK to leave it for now. But if you want, feel free to fix that in this PR :)

Reply:
I'll try fix it too :))

Reply:
@rintaro Just updated removing the `error: expected pattern` too 👍